lib/linguist/strategy/extension.rb

```diff
@@ -2,8 +2,21 @@ module Linguist
   module Strategy
     # Detects language based on extension
     class Extension
-      def self.call(blob, _)
-        Language.find_by_extension(blob.name.to_s)
+      # Public: Use the file extension to detect the blob's language.
+      #
+      # blob               - An object that quacks like a blob.
+      # candidates         - A list of candidate languages.
+      #
+      # Examples
+      #
+      #   Extension.call(FileBlob.new("path/to/file"))
+      #
+      # Returns an Array of Languages associated blob's file extension.
```

Comment:
As above regarding case, and we need a few more words… “...associated with a blob’s file extension.”